source/github-helpers/heat-map.css

```diff
@@ -4,42 +4,56 @@ Original file: https://github.githubassets.com/assets/code-d21b20a03009.css
 These colors are for normally-gray text.
 */
 
+:root {
+	--rgh-heat-color-deg: 29deg;
+	--rgh-heat-color-1: hsl(var(--rgh-heat-color-deg) 100% 38%);
+	--rgh-heat-color-2: hsl(var(--rgh-heat-color-deg) 69% 40%);
+	--rgh-heat-color-3: hsl(var(--rgh-heat-color-deg) 57% 41%);
+	--rgh-heat-color-4: hsl(var(--rgh-heat-color-deg) 47% 41%);
+	--rgh-heat-color-5: hsl(var(--rgh-heat-color-deg) 36% 42%);
+	--rgh-heat-color-6: hsl(var(--rgh-heat-color-deg) 26% 43%);
+	--rgh-heat-color-7: hsl(var(--rgh-heat-color-deg) 15% 44%);
+	--rgh-heat-color-8: hsl(var(--rgh-heat-color-deg) 6% 44%);
+	--rgh-heat-color-9: hsl(var(--rgh-heat-color-deg) 4% 45%);
+	--rgh-heat-color-10: hsl(var(--rgh-heat-color-deg) 2% 45%);
```

Comment:
Only one variable is needed: --rgh-heat-color-deg. I don't want to add this huge list of variables

Reply:
But these variables make it possible to overwrite individual values if desired

Reply:
I don't want to offer this kind of granularity in Refined GitHub, I'm only allowing a single variable because it's simple enough. All other cases are already covered by custom selectors. 

Keeping up with GitHub's constant changes is hard as it is and I want to keep everything else as simple as possible.